File: VB/WindowsFormsApplication82/CustomRibbonControl.vb
Imports DevExpress.XtraBars.Ribbon
Imports DevExpress.XtraBars.Ribbon.ViewInfo
Imports System
Imports System.Collections.Generic
Imports System.Drawing
Imports System.Linq
Imports System.Text

Namespace WindowsFormsApplication82
	Public Class CustomRibbonControl
		Inherits RibbonControl

		Private _captionColor As Color

		Public Sub New()
			MyBase.New()

		End Sub
		Protected Overrides Function CreateViewInfo() As RibbonViewInfo
			Return New CustomRibbonViewInfo(Me)
		End Function
		Public Property CaptionColor() As Color
			Get
				Return _captionColor
			End Get
			Set(ByVal value As Color)
				_captionColor = value
			End Set
		End Property
	End Class
End Namespace

File: CS/WindowsFormsApplication82/CustomRibbonControl.cs
using DevExpress.XtraBars.Ribbon;
using DevExpress.XtraBars.Ribbon.ViewInfo;
using System;
using System.Collections.Generic;
using System.Drawing;
using System.Linq;
using System.Text;

namespace WindowsFormsApplication82 {
    public class CustomRibbonControl : RibbonControl {
        private Color _captionColor;

        public CustomRibbonControl() : base() {

        }
        protected override RibbonViewInfo CreateViewInfo() {
            return new CustomRibbonViewInfo(this);
        }
        public Color CaptionColor { get { return _captionColor; } set { _captionColor = value; } }
    }
}
